jobs/ci-run/integration/gen/test-model.yml

```diff
@@ -32,21 +32,9 @@
     - multijob:
         name: 'IntegrationTests-model'
         projects:
-        - name: 'test-model-test-model-config-lxd'
+        - name: 'test-model-test-model-destroy-lxd'
```

Comment:
Is there a particular reason to use the naming pattern `test-model-test-model-*-*` instead of `test-model-*-*`

Reply:
this autogeneration.

`test-model-` means model suite.
`-test-model-destroy` means test-model-destroy tests in 'model suite'.

Reply:
Probably a bit nitty, but if the suite is for testing models (`test-model`), there is no reason to repeat again `test-model` in the test, that is implicit in the suite. If that's the general approach we are following, I'm OK with that.